cherrysrv/commands.go

```diff
@@ -284,7 +307,7 @@ func do_login(clt *Client, args string) {
 
 	/* Update player */
 
-	clt.Say("/login>0>you're now %s", clt)
+	clt.Say(">/login>0>you're now %s", clt)
```

Comment:
Consistency change.

Reply:
will apply it directly, thanks for catching it!